### PR TITLE
Fix some more warnings

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: none
affects: none
release note: none

### CODE REVIEW

File: Canvas/Canvas.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3216,7 +3216,6 @@
 				F9B947052215E97A0083E660 /* Carthage Copy Frameworks */,
 				3B6DCCB51FB502CE005EE8D2 /* Embed App Extensions */,
 				4902E1271CF4CBA00069E043 /* Fabric */,
-				B186F36C2144BE340026429F /* Strip Frameworks */,
```
Comment (vanwagonet) [inline Canvas/Canvas.xcodeproj/project.pbxproj]: Strip frameworks seemed to be a Realm thing we don't need anymore.

File: Canvas/CanvasKit1/Models/CKDiscussionEntry.m [hunk 2]
```diff
@@ -214,8 +214,6 @@ - (NSString *)JSONString
         }
     }
     
-    [attachmentInfoArray sortUsingSelector:@selector(compareAttachmentsIndex:)];
```
Comment (vanwagonet) [inline Canvas/CanvasKit1/Models/CKDiscussionEntry.m]: `compareAttachmentsIndex` doesn't exist.

File: Canvas/CanvasKit1/UI Classes/CKCommentViewController.m [hunk 3]
```diff
@@ -211,28 +211,6 @@ - (void)didReceiveMemoryWarning {
     // Release any cached data, images, etc that aren't in use.
 }
 
-- (void)viewWillUnload {
```
Comment (vanwagonet) [inline Canvas/CanvasKit1/UI Classes/CKCommentViewController.m]: these are deprecated, and not called anymore.

File: Frameworks/CanvasKeymaster/CanvasKeymaster/Domain Picker/CKMLocationSchoolSuggester.m [hunk 4]
```diff
@@ -23,7 +23,6 @@
 @import Mantle;
 @import CanvasKit;
 
-static double const CKMDistanceThreshold = 50.0;
```
Comment (vanwagonet) [inline Frameworks/CanvasKeymaster/CanvasKeymaster/Domain Picker/CKMLocationSchoolSuggester.m]: unused variable

File: Frameworks/CanvasKit/CanvasKit.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -1442,7 +1442,7 @@
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				FRAMEWORK_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(PROJECT_DIR)/../Carthage/Build/iOS/**",
+					"$(PROJECT_DIR)/../../Carthage/Build/iOS/**",
```
Comment (vanwagonet) [inline Frameworks/CanvasKit/CanvasKit.xcodeproj/project.pbxproj]: fixed relative path

File: Student/Student.xcodeproj/project.pbxproj [hunk 6]
```diff
@@ -734,7 +731,6 @@
 				78761D85218B8C2A0042E5D2 /* StudentShare */,
 				2C964862211E37EC002C3255 /* Frameworks */,
 				2C5C698A211DFFAD00D9AD15 /* Products */,
-				5B22A366219812380002DBD0 /* Recovered References */,
```
Comment (vanwagonet) [inline Student/Student.xcodeproj/project.pbxproj]: weird extra references to Core?

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 90.48% | 90.41% | 0.07%
**Student** | 94.52% | 94.52% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.3% | 92.3% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**